s3/replication/replicator/tests/system/data/replication_job.json

```diff
@@ -0,0 +1,34 @@
+{
+  "replication-id": "bucketname-objname-versionid-objcreatetime",
+  "replication-event-create-time": "2021-05-05T08:10:14.000Z",
+  "source" : {
+    "endpoint": "http://s3.seagate.com",
+    "accesss_key": "accesss_key value",
+    "secret_key": "secret_key value",
+    "operation": {
+      "type": "replicate_object",
+      "attributes": {
+        "ACL": "base64-encoded-acl-json",
+        "Bucket-Name": "sourcebucket",
+        "Object-Name": "seagate-hdd.jpg",
+        "Content-Length": "0",
+        "Content-MD5": "d41d8cd98f00b204e9800998ecf8427e",
+        "Content-Type": "",
+        "Date": "2021-05-05T08:10:14.000Z",
+        "Last-Modified": "2021-05-05T08:10:14.000Z",
+        "Owner-Account": "cloud",
+        "Owner-Account-id": "708741171239",
+        "Owner-Canonical-id": "f5810cae61af4d8d99b113cdc3c55443c7f6eb36fec041f88b05f9704daeed47",
+        "Owner-User": "root",
+        "Owner-User-id": "BJo5ILHCTOiXYrWrkitJeg",
+        "x-amz-version-id": "MTg0NDY3NDI0NTM1MDczNDEwMjU",
+      }
+    }
+  },
+  "target": {
+    "endpoint": "http://s3.seagate.com",
+    "accesss_key": "accesss_key value",
+    "secret_key": "secret_key value",
+    "bucket-name": "targetbucket"
```

Comment:
can we go for Bucket-Name ???
we have used this key name in in source attributes & same for other places.

Reply:
Done